_data/staff.yaml

```diff
@@ -1,70 +1,3 @@
-  description: Hi everyone! My name is Brian and I'm a senior CS major from Cleveland, Ohio. Feel free to hit me up for boba or if you want to talk about our Lord and Savior, LeBron James
-
-- name: Ian McDonald
-  title: Facilitator
-  img: images/ian.jpg
-  description: I’m a senior studying computer science. I started using Linux after taking some bad advice and I haven’t looked back since.
-
-- name: Harrison Kuo
-  title: Facilitator
-  img: images/kuoh.png
-  description: I'm a senior studying EECS/MSE and I like messing with old or broken computers on my free time. I'm also a deputy manager at the OCF.
-
-- name: Jason Perrin
-  title: Facilitator
-  img: images/jvperrin.jpg
-  description: I'm a junior EECS major who enjoys security, systems, webcomics, and running, though hopefully not running away from angry decal students!
-
-- name: Kevin Kuehler
-  title: Facilitator
-  img: images/keur.jpg
-  description: I'm a junior CS transfer student from San Diego. Using Linux is what got me interested in computer science, and I'm excited to share that interest with others!
-
-- name: Daniel Kessler
-  title: Facilitator
-  img: images/dkessler.png
-  description: I'm a junior studying computer science and economics. I've been messing around with Linux servers for years, and I'm excited to help out with this DeCal!
-
-- name: Tony Situ
-  title: Facilitator
-  img: images/tonyc2.jpg
-  description: California born and raised; chasing whatever catches my curiosity at the time. On the side, I love tabletop, hiking, photography, and value (especially value).
-
-- name: Loren McIntyre
-  title: Facilitator
-  img: images/mcint.jpg
-  description: I'm a 4th-year CS major who took some time off along the way. I like to talk about software philosophy, life, and living in the co-ops.
-
-- name: Xufei Tan
-  title: Facilitator
-  img: images/tanx.png
-  description: I'm a sophomore CS student from Pennsylvania who enjoys reading, playing League, and boba. I'm still learning about Unix, so tell me about anything cool you found!
-
-- name: Sahil Hasan
-  title: Facilitator   
-  img: images/shasan.png  
-  description: I'm a junior EECS major that spends far too much time in the OCF or playing CS:GO, not that those two are mutually exlusive. 
-
-- name: Mathew Cha
-  title: Facilitator
-  img: images/mdcha.png
-  description: I'm a 3rd-year Math / CS student from SoCal. In my free time, I play board games, League, and cook!
-
-- name: Benjamin Zhang
-  title: Facilitator
-  img: images/bzh.jpg
-  description: I'm a freshman studying Astrophysics and CS. My favorite hobby is sleeping, my second favorite is falling asleep, and my third favorite is playing board games.
-
-- name: Minos Park
-  title: Facilitator
-  img: images/minos.png
-  description: Berkeley is my spritual home. Fell in love with this place whiling living in the Bay Area for 5+ years now. Also Orange, Lee, 180-29-8724.
-  
+  img: https://static.ocf.berkeley.edu/img/penguin-swing.svg
```

Comment:
I'd just leave the few people you know are going to stay facilitators this semester (maybe get new bios/pictures later though), instead of adding placeholder content here.

Reply:
I removed it so new facilitators get the opportunity to contribute a commit to the codebase and hopefully become more comfortable developing for the decal